service-runner.js

```diff
@@ -69,13 +69,13 @@ ServiceRunner.prototype.run = function run(conf) {
 
         // Set up the logger
         if (!config.logging.name) {
-            config.logging.name = name;
+            config.logging.name = (config.services && config.services.name) || name;
```

Comment:
`config.services` is an Array. But. this is not the right place nor way to set the name. [`_runWorker()`](https://github.com/wikimedia/service-runner/blob/master/service-runner.js#L278-L285) conveys that info to individual services and already sets the name.